Installing 4.5: add IBM Z and IBM Power, both normal and restricted network versions

rh-max opened the pull request.

ahardin-rh commented on installing/installing_ibm_z/installing-ibm-z.adoc:
```diff
@@ -76,6 +81,12 @@ include::modules/installation-registry-storage-non-production.adoc[leveloffset=+
+
+.Additional resources
+
+* See also https://access.redhat.com/solutions/4387261[How to generate SOSREPORT within OpenShift4 nodes without SSH].
```
link:https://access.redhat.com/solutions/4387261[How to generate SOSREPORT within OpenShift4 nodes without SSH].

ahardin-rh commented on modules/installation-approve-csrs.adoc:
```diff
@@ -32,37 +32,34 @@ these CSRs are approved or, if necessary, approve them yourself.
-endif::ibm-z[]
-ifndef::ibm-z[]
 ----
 $ oc get nodes
```
We're now creating one code block for the command itself and a separate code block for its output: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks

That style update came with the introduction of the copy/paste clipboard feature on docs.openshift.com

ahardin-rh commented on modules/installation-approve-csrs.adoc:
```diff
@@ -78,6 +75,16 @@ csr-c57lv   5m26s   system:node:ip-10-0-95-157.us-east-2.compute.internal
+endif::ibm-z[]
+ifdef::ibm-z[]
+----
+$ oc get csr
```
We'll need two separate code blocks for the command and its output here too.

ahardin-rh commented on modules/installation-complete-user-infra.adoc:
```diff
@@ -26,6 +29,24 @@ cluster on infrastructure that you provide.
+$ oc edit configs.samples cluster
+----
++
+Then, change the line
```
I would consider restructuring this step to be part of the numbered procedure.

    . Change the line `managementState: Removed` to `managementState: Managed`.

rh-max replied:
This step needs to remain under the conditional of "if you are not installing, but upgrading". If I promote it to being a separate step, it will escape that conditional.

ahardin-rh replied:
@rh-max Sounds good. Thanks!

ahardin-rh commented on modules/installation-registry-storage-non-production.adoc:
```diff
@@ -36,3 +36,23 @@ Error from server (NotFound): configs.imageregistry.operator.openshift.io "clust
+$ oc edit configs.imageregistry/cluster
+----
++
+Then, change the line
```
I would consider restructuring this step to be part of the numbered procedure.

    . Change the line `managementState: Removed` to `managementState: Managed`.

rh-max replied:
Same thing here: I don't have a clean way to separate the "change the line" step while keeping it under the scope of "Ensure that your registry is set to managed...".

ahardin-rh commented:
@rh-max Nicely done :+1: Just some minor comments from me. :rocket:

rh-max commented:
@ahardin-rh Thanks! I have implemented your suggestions but for the last two. If you disagree with my reasoning and maintain that it should be changed, please leave the corresponding replies. If it's fine as it is, I think Vikram or Cody should take over from here (so you don't need to merge it).

@vikram-redhat @codyhoag I've added the release notes in
https://github.com/openshift/openshift-docs/pull/24254/commits/8ec64471e62e54678f22ab3b1803d11b2c180056
, could you PTAL? Note the placeholder title on line 2022, I don't know where to get the requisite information.

ahardin-rh commented:
@rh-max Once all edits are in, we'll need to squash the commits down to one. Thanks again!

vikram-redhat commented:
Thanks @rh-max. It might need a live id assigned.
@codyhoag @jeana-redhat - when you are in, do you know if live id for https://github.com/openshift/openshift-docs/commit/8ec64471e62e54678f22ab3b1803d11b2c180056#diff-fd66196c764f46c6b4217c76d3c628b0R2022 has been assigned?

jeana-redhat commented:
@vikram-redhat Is it just the bugfix advisory for the release? If so, that one is RHBA-2020:3028

codyhoag commented:
@vikram-redhat @jeana-redhat TL;DR I agree with Jeana :slightly_smiling_face: 

If we're following what we've done in the past, it should be the bug fix advisory and would follow our standard formatting/naming conventions (4.3 example with IBM Z/Power additions is [here](https://docs.openshift.com/container-platform/4.3/release_notes/ocp-4-3-release-notes.html#ocp-4-3-18)). With that said, `RHBA-2020:3028` would be the one to use here.

rh-max commented:
@jeana-redhat @codyhoag I've added the ID to the title. The release notes part of the PR is ready for review, could one of you PTAL?
Besides, I've had a go at the introductory paragraph with what information I had. Let me know what needs to be changed in it or if it even needs to be there. Thanks!

jeana-redhat commented:
@rh-max as far as the intro goes, there's a bit more to it but I have it in the work I'm doing - I think it's ok to leave that how you have it and I'll resolve the merge conflict as needed

jeana-redhat commented:
Should have said - RN stuff LGTM, unless @codyhoag has concerns I'm missing.

codyhoag commented on release_notes/ocp-4-5-release-notes.adoc:
```diff
@@ -2068,3 +2068,95 @@ was not allowed to use local images. The container image signature policy has
+link:https://access.redhat.com/errata/RHBA-2020:3028[RHBA-2020:3028] advisory.
+
+
+[[id="ocp-4-5-4-features"]]
```
These IDs are not rendering correctly. I believe when using the `id=` version, it should only be accompanied with one bracket:
```
[id="ocp-4-5-4-features"]
```

codyhoag commented on release_notes/ocp-4-5-release-notes.adoc:
```diff
@@ -2068,3 +2068,95 @@ was not allowed to use local images. The container image signature policy has
+==== Features
+
+
+[[id="ocp-4-5-4-ibm-z"]]
```
Same here: `[id="ocp-4-5-4-ibm-z"]`

codyhoag commented on release_notes/ocp-4-5-release-notes.adoc:
```diff
@@ -2068,3 +2068,95 @@ was not allowed to use local images. The container image signature policy has
+** HyperPAV enabled on IBM System Z for the virtual machine for FICON attached ECKD storage.
+
+
+[[id="ocp-4-5-4-ibm-power"]]
```
And here: `[id="ocp-4-5-4-ibm-power"]`

vikram-redhat commented:
@rh-max when you are in next, let's make sure we sync up on this content and making it live.

rh-max commented:
@jeana-redhat @codyhoag Thanks! Fixed the IDs and left a comment near my version of introduction.

rh-max commented:
This PR has ACKs both on the main docs part (not the release notes) and on the release notes. So it's ready to be merged once the release happens, @vikram-redhat .
